inc/settings.php

```diff
@@ -909,6 +909,29 @@ function siteorigin_north_menu_breakpoint_css() {
 endif;
 add_action( 'wp_head', 'siteorigin_north_menu_breakpoint_css' );
 
+if ( ! function_exists( 'siteorigin_north_sidebar_zero_css' ) ) :
+/**
+ * Add CSS when sidebar width is 0
+ */
+function siteorigin_north_sidebar_zero_css() {
+
+	$breakpoint = siteorigin_setting( 'structure_sidebar_width' );
+
+	$css = '<style type="text/css" id="siteorigin-sidebar-css">' . "\t" .
+	'/* Sidebar fix */' . "\t" .
+	'.site-content .widget-area {' .
+		'display: none;' .
+	'}' . "\t" .
+	'</style>';
+
+	if( siteorigin_setting( 'structure_sidebar_width' ) == 0 ) {
+		echo $css;
+	}
+
+}
+endif;
+add_action( 'wp_head', 'siteorigin_north_sidebar_zero_css' );
```

Comment:
Rather than adding a separate `<style>` to the head, in future, we can take advantage of the `siteorigin_settings_custom_css` filter. This way we'll be adding all CSS to the main custom CSS style tag.

I modified this filter to also include the settings array - https://github.com/siteorigin/settings/commit/41920271a883d5746358e95d889a0ba61c535857